source/DataBase/DataBaseCommon.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <vector>
+#include <string>
+struct DataBaseCommon
+{
+    static const std::vector<std::string> userRecordsRecords;
+    static const std::vector<std::string> categoriesRecords;
+    static const std::vector<std::string> passwordHistoryRecords;
+    static const std::vector<std::string> passwordRecords;
+
+    static const std::string dataBasePath;
+
+    static const std::string users;
+    static const std::string passwords;
+    static const std::string passwordHistory;
+    static const std::string categories;
+};
+
+const std::vector<std::string> DataBaseCommon::userRecordsRecords{"Username", "Password"};
+const std::vector<std::string> DataBaseCommon::categoriesRecords{"Category"};
+const std::vector<std::string> DataBaseCommon::passwordHistoryRecords{"Creation_Time", "Modify_Time", "Expiry_Time"};
+const std::vector<std::string> DataBaseCommon::passwordRecords{
+    "User_Id", "Category_Id", "Title", "Username", "Password", "Note", "Url"};
```

Comment:
The same reason as above.
```namespace database
{
    namespace common
    {
        static const std::vector<const char *> userRecordsRecords{"Username", "Password"};
    }
}
```

Reply:
maybe I'll leave this std::vector<std::string>? I use many string functionalities in these functions.

Many of these methods may change with the DataBaseManagment task